charts/datadog/templates/_container-otel-agent.yaml

```diff
@@ -51,6 +51,8 @@
       value: "5009"
     - name: DD_AGENT_IPC_CONFIG_REFRESH_INTERVAL
       value: "60"
+    - name: DD_OTELCOLLECTOR_ENABLED
```

Comment:
this will set env var to boolean `true` instead of boolean string, need to use `quote` or only add this envvar when it's enabled (assuming it's disabled by default in otel agent).

Reply:
Good catch, yes by default this is disabled, so I change to only add the env var when this config is enabled